<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogInfo</class>
 <widget class="QDialog" name="DialogInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbPosX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Posicion X:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPosY">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Posicion Y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbVel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Velocidad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbId">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Id:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
